src/prof.c

```diff
@@ -310,6 +310,7 @@ prof_leave(tsd_t *tsd, prof_tdata_t *tdata) {
 }
 
 #ifdef JEMALLOC_PROF_LIBUNWIND
+JEMALLOC_ALIGNED(CACHELINE)
```

Comment:
Oh I intended to add it in a separate diff. Stashed here for benchmarking but forgot to remove... Let's keep it since it's needed anyway?